espnet/nets/pytorch_backend/e2e_asr_transducer.py

```diff
@@ -282,7 +283,11 @@ def forward(self, xs_pad, ilens, ys_pad):
             batch_nbest = []
 
             for b in six.moves.range(batchsize):
-                nbest_hyps = self.dec.recognize_beam(hs_pad[b], self.recog_args)
+                if self.beam_size == 1:
```

Comment:
ditto

Reply:
ditto. This code applies only to training.